src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -116,33 +116,34 @@ def apply(self, organizations, parameter_store, config):  # pylint: disable=R091
                     )
                     if stored_policy not in _policies:
                         OrganizationPolicy.clean_and_remove_policy_attachment(
-                            organization_mapping, path, organizations, _type)
+                            organization_mapping, path, policy_filename, organizations, _type)
             except ParameterNotFoundError:
                 LOGGER.debug(
                     'Parameter %s was not found in Parameter Store, continuing.', policy)
                 pass
 
             for _policy in _policies:
-                path = OrganizationPolicy._trim_scp_file_name(
-                    _policy) if policy == 'scp' else OrganizationPolicy._trim_tagging_policy_file_name(_policy)
+                path, policy_filename = OrganizationPolicy._trim_policy_file_name(
+                    _policy)
+                policy_name = 'adf-{0}-{1}--{2}'.format(policy, path, policy_filename) if policy_filename != '{}.json'.format(policy) else 'adf-{0}-{1}'.format(policy, path)
```

Comment:
This would become a function call again.

Reply:
Fixed in recent commit.